test/e2e/job_scheduling.go

```diff
@@ -604,7 +513,7 @@ var _ = Describe("Job E2E Test", func() {
 		if expectPod%1 == 1 {
 			expectPod--
 		}
-		err = wait.Poll(100*time.Millisecond, oneMinute, func() (bool, error) {
+		err = wait.Poll(100*time.Millisecond, twoMinute, func() (bool, error) {
```

Comment:
Why change this, how come the scheduling is so slow?

Reply:
It seems to be releated the cluster environment at that time.It will be timeout in rare cases. I've tried serval times and it woks well after set the timeout to be 2 minutes.